lib/tasks/db.rake

```diff
@@ -1,3 +1,5 @@
+require 'ostruct'
```

Comment:
is this being used?

Reply:
No - I was using it as an interim thing I'll remove.

Reply:
cool